README.md

```diff
@@ -21,7 +21,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+That is a migration file. Migration files are used to alter the database schema so that we don't need to write SQL by hand. The number in front of the file tells you when the migration file was created.
```

Comment:
:+1:

What change, to our database schema, would you expect from this migration?